test/k8sT/fqdn.go

```diff
@@ -48,24 +48,27 @@ var _ = Describe("K8sFQDNTest", func() {
 		// In case the IPs changed from above, update them here
 		var lookupErr error
 		err := helpers.WithTimeout(func() bool {
-			var addrs []string
-			addrs, lookupErr = net.LookupHost("vagrant-cache.ci.cilium.io")
-			if lookupErr != nil {
-				lookupErr = fmt.Errorf("error looking up vagrant-cache.ci.cilium.io: %s", lookupErr)
+			addrs, err2 := net.LookupHost("vagrant-cache.ci.cilium.io")
```

Comment:
Why not use google.com or something similar with high availability guarantees?

Reply:
I don't think changing the domain name we request will have any impact on the success of DNS requests. We already use Google's DNS, if I recall correctly, to resolve domain names.
The problem is likely Packet's connection to the outside, so any DNS resolver will have the same issues.

Reply:
Hence a longer timeout may help to carry over intermittent connectivity issues.

Reply:
That DNS resolution relies on `getaddrinfo(3)` under the hood. I don't think `getaddrinfo(3)` implements any retry. I couldn't find any retry mechanism at the Golang layer either.

In case of intermittent connectivity issues, the request packet won't stay around, it will be dropped. So I don't think waiting longer for an answer is going to solve this.